gemrb/plugins/GUIScript/GUIScript.cpp

```diff
@@ -66,8 +66,10 @@
 #include <algorithm>
 #include <cstdio>
 
+#if !defined(WIN32) && !defined(__MINGW32__)
 // MIPSPro fix for IRIX
 size_t strlcpy(char *, const char *, size_t);
+#endif
```

Comment:
On MinGW, it's decorated by some `__declspec(dllimport)`, so this redef. breaks.

Reply:
I think it would be nicer to just check in cmake and then use a specific define. Look at the main file and cmake/cmake_config.h.in

Reply:
Ok.